assets/src/edit-story/masks/test/output.js

```diff
@@ -125,41 +125,41 @@ describe('WithMask', () => {
     await expect(content).toContain(MaskTypes.HEART);
   });
 
-  // see https://github.com/google/web-stories-wp/issues/536
-  // eslint-disable-next-line jest/no-disabled-tests
-  it.skip('should produce valid AMP output', async () => {
-    const props = {
-      element: {
-        id: '123',
-        type: 'image',
-        mimeType: 'image/png',
-        scale: 1,
-        origRatio: 9 / 16,
-        x: 50,
-        y: 100,
-        height: 1920,
-        width: 1080,
-        rotationAngle: 0,
-        resource: {
+  describe('AMP validation', () => {
+    it('should produce valid AMP output', async () => {
+      const props = {
+        element: {
+          id: '123',
           type: 'image',
           mimeType: 'image/png',
-          src: 'https://example.com/image.png',
+          scale: 1,
+          origRatio: 9 / 16,
```

Comment:
I don't think this exists anymore, I'd confirm

Reply:
Yup, good catch!

Need to find a good way to catch these things...